test/withStyles_test.jsx

```diff
@@ -196,7 +198,12 @@ describe('withStyles()', () => {
       }))(MyComponent);
 
       // copied
-      expect(Wrapped.propTypes).to.eql(MyComponent.propTypes);
+      const expectedPropTypes = deepmerge({}, MyComponent.propTypes);
+      delete expectedPropTypes.styles;
+      delete expectedPropTypes.theme;
+      expect(Wrapped.propTypes).to.eql(expectedPropTypes);
+      expect(MyComponent.propTypes).to.include.keys('styles', 'theme');
```

Comment:
Doesn't the assertion on 204 already handle that? Or do I not understand how `eql` treats objects?